packages/theme-context/src/ThemeContext/index.test.js

```diff
@@ -7,17 +7,17 @@ describe("themes/ThemeContext/index", () => {
     {
       name: "default",
       value: expect.objectContaining({
-        Consumer: expect.any(Function),
-        Provider: expect.any(Function)
+        Consumer: expect.anything(Function),
```

Comment:
Anything is not the same as any, these should still be any.

`expect.any(constructor) matches anything that was created with the given constructor`
`expect.anything() matches anything but null or undefined`